tern/command_lib/snippets.yml

```diff
@@ -121,3 +121,9 @@ yarn:
     - 'add'
     - 'remove'
   packages: 'npm'
+zypper:
```

Comment:
Add a space above this line to be consistent with formatting.

Reply:
Done!

Reply:
Looks like a bunch of other commits got included with this one. Can you rebase from current tip of master?